### PR TITLE
2d controlnet tutorial

### LINKED ISSUE #362: Add tutorial for ControlNets
Add tutorial training a diffusion model on brain 2D images from the BRATS dataset. To obtain the 2D images, we could use the data tranformations from the anomaly detections [examples](https://github.com/Project-MONAI/GenerativeModels/blob/main/tutorials/generative/anomaly_detection/2d_classifierfree_guidance_anomalydetection_tutorial.ipynb). After having a model trained, train a controlnet that learns how to generate a brain given the bain mask. In this case, I guess it will be necessary just to fine tuning the controlnet paramenters (it might not be necessary to change the original diffusion model). 
Finally, evaluate the performance of the model by generating the brains using the masks from the the validation set.

An possible extension, is show the use multicontrolnets and finetune a second network to generate brains given tumours. Then, in the evaluation combine the 2 controlnets (brain mask + tumour mask) to generate an image.

### PR DESCRIPTION
**Pull request by marksgraham**
Fixes #362

### CODE REVIEW

**Review by Warvito — APPROVED** [review]
Thanks, @marksgraham ! It is cool to see how well the conditioning works!
